setup.py

```diff
@@ -1,16 +1,18 @@
 from setuptools import setup, find_packages
 setup(
     name="Momoi Bot",
-    version="0.0.1",
+    version=open("discord_bot/_version.py").readlines()[-1].split()[-1].strip("\"'"),
```

Comment:
Why don't we just throw `from _version import __version__` at the top of the file?